tracer/test/Datadog.Trace.Security.Unit.Tests/WafObfuscationTests.cs

```diff
@@ -0,0 +1,77 @@
+// <copyright file="WafObfuscationTests.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Specialized;
+using System.Linq;
+using Datadog.Trace.AppSec;
+using Datadog.Trace.AppSec.Waf;
+using Datadog.Trace.AppSec.Waf.ReturnTypes.Managed;
+using Datadog.Trace.Configuration;
+using Datadog.Trace.Vendors.Newtonsoft.Json;
+using FluentAssertions;
+using Xunit;
+
+namespace Datadog.Trace.Security.Unit.Tests
+{
+    [Collection("WafTests")]
+    public class WafObfuscationTests
+    {
+        [Theory]
+        [InlineData(false, "bearer", "this is a very secret value having the select pg_sleep attack", "select pg_sleep")]
+        [InlineData(false, "password", "select pg_sleep", "select pg_sleep")]
+        [InlineData(false, "pwd", "select pg_sleep", "select pg_sleep")]
+        [InlineData(true, "bearer", "this is a very secret value having the select pg_sleep attack", "select pg_sleep")]
+        [InlineData(true, "password", "select pg_sleep", "select pg_sleep")]
+        [InlineData(true, "pwd", "select pg_sleep", "select pg_sleep")]
+        public void AttacksWithSecrets(bool obfuscate, string key, string fullAttack, string highlight)
+        {
+            Environment.SetEnvironmentVariable("DD_TRACE_DEBUG", "true");
```

Comment:
I don't think we should set this here if possible, as this is changing the global environment variables, and could conflict with other tests? Doesn't look required AFAICT

Reply:
To be honest, I just copied and pasted it from another WAF test. I agree, it probably going to slow things and could create issues with other tests, so I the line from both places in 15eff6708.

Reply:
The debug variable is still being set here. Maybe you committed before saving this file?

Reply:
Okay, really fixed in c66cff586. Yes, can't have saved the file.